### PR TITLE
Fix e2e test for mozart failover

### PR DESCRIPTION
**Pull request by PriyaKR**
Resolves [https://github.com/bbc/simorgh-infrastructure/issues/424]

**Overall change:** _Added a test to assert whether the article page is served from mozart cache._

**Code changes:**

- _Updated the statusSpec.js with a test to assert for mozart fallback value in header response which is set to a value of 1 when there is fail over ._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by PriyaKR** [comment]
Have added the test under the statusSpec.js wondering whether this is something that will go under dataSpec.js as there is a test in dataSpec.js which checks for 200 status code for the page.

**Review by DemonusPC — DISMISSED** [review]
dank

File: package-lock.json [hunk 1]
```diff
@@ -6355,7 +6355,8 @@
     "detect-libc": {
       "version": "1.0.3",
       "resolved": "https://registry.npmjs.org/detect-libc/-/detect-libc-1.0.3.tgz",
-      "integrity": "sha1-+hN8S9aY7fVc1c0CrFWfkaTEups="
+      "integrity": "sha1-+hN8S9aY7fVc1c0CrFWfkaTEups=",
```
Comment (sareh) [inline package-lock.json]: Could you please ensure you're running node version `10.15.3` & then re-run the `npm install` - You should see that then these `optional true` changes aren't needed on the branch.

File: package-lock.json [hunk 2]
```diff
@@ -6355,7 +6355,8 @@
...
+      "optional": true
```
Comment (sareh) [inline package-lock.json]: Could you please ensure you're running node version `10.15.3` & then re-run the `npm install` - You should see that then these `optional true` changes aren't needed on the branch.

Comment (PriyaKR) [reply package-lock.json]: The file difference doesn't seem to show the change in`package-lock.json` file.

**Review by chris-hinds — DISMISSED** [review]
Looks good to me 👍 Thanks Priya

**Review by DemonusPC — DISMISSED** [review]
gg wp

**Comment by PriyaKR** [comment]
Me and @hindsc52 have tested whether e2e tests are failing when there is mozart failover and it did actually fail when we killed the mozart application on test.

**Review by jamesdonoh — DISMISSED** [review]
Looks good, @PriyaKR I tried it locally and I get the same `package-lock` changes so I think it's fine?

**Review by chris-hinds — DISMISSED** [review]

**Review by rossgaskell — APPROVED** [review]
👍

**Review by chris-hinds — APPROVED** [review]
👍

**Comment by PriyaKR** [comment]
Tested already nothing has changed since then other merging the latest commits to this branch.

**Pull request merged**